Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollingCoordinator.mm

```diff
@@ -185,6 +186,11 @@
     return WheelEventHandlingResult::handled();
 }
 
+void RemoteScrollingCoordinator::scrollingTreeNodeScrollbarVisibilityDidChange(uint64_t nodeID, WebCore::ScrollbarOrientation orientation, bool isVisible)
```

Comment:
I don't think you need this override at all?

Reply:
Since this needs to be a RemoteScrollingCoordinator function for the IPC, I dont think there’s a way to get around having this function.

Reply:
OK. The first argument should be a ScrollingNodeID.

Reply:
Fixed.